### PR TITLE
Integrate App ID for account management

### PR DESCRIPTION
**Pull request by data-henrik**

### CODE REVIEW

File: .bluemix/pipeline-DEPLOY.sh [hunk 1]
```diff
@@ -10,46 +12,51 @@ fi
...
-  ibm:yp:eu-de)
-  domain=".eu-de.mybluemix.net"
-  ;;
-esac
-if ! cf app $CF_APP; then
-  cf push $CF_APP -n $CF_APP -f $MANIFEST --no-start
+# Create services
+bx service create conversation free insurance-bot-conversation
+bx service create cloudantNoSQLDB Lite ${PREFIX}insurance-bot-db
+bx service create appid "Graduated tier" insurance-bot-appid
```
Comment (l2fprod) [inline .bluemix/pipeline-DEPLOY.sh:18]: @data-henrik 

we had ${PREFIX} for the db to make sure we did not mix the dev db with the master db so both deployments are isolated.

Any impact having dev+master using the same App ID?

I'm surprised we use the same conversation service - if I change the conversion flow in dev, master will not know how to work with the updated workspace given it is shared

Comment (data-henrik) [reply .bluemix/pipeline-DEPLOY.sh:18]: Ideally, we have different instances for all three services. But that is extra to the integration. How about we address that with a separate defect?

**Comment by VidyasagarMSC** [comment]
screen shot 2018-07-06 at 2 44 06 pm

Looks like everything is working except I don't see it navigating to the next screen.

**Review by VidyasagarMSC — COMMENTED** [review]
Everything looks working except the signup and login and not navigating to the next screen.

**Comment by VidyasagarMSC** [comment]
Need all this to be documented. Approving this

**Pull request merged**